src/main/java/org/llorllale/cactoos/matchers/MatcherOf.java

```diff
@@ -46,9 +46,7 @@
  *
  * @param <T> Type of object to match
  * @since 0.12
- * @todo #135:30min Remove all constructors except the last one so that
- *  every matcher implemented using MatcherOf take care of properly
- *  describe itself and the mismatch.
+ * @todo #204:30m/DEV Rewrite the Matcher's, which constructed from others constructor, to use the last constructor of MatcherOf. And delete all constructors except last one.
```

Comment:
@ruca905 I don't think the `#204:30m/DEV` format is correct, I think it should be `#165:30min` because you are solving the issue #165, not #204!